coderd/inboxnotifications_test.go

```diff
@@ -0,0 +1,288 @@
+package coderd_test
```

Comment:
The coverage of `inboxnotifications.go` is only about 41%. The two major testing blindspots right now are:
- error conditions in general
- all of `watchInboxNotifications`